### ISSUE TITLE
problems when running mnist-keras example

### ISSUE THREAD

**Issue by DoLNw** [HewlettPackard/swarm-learning#59]
when I try to start training, I met following problems

<img width="1381" alt="Screen Shot 2021-10-11 at 7 19 27 PM" src="https://user-images.githubusercontent.com/35894540/136781762-63159581-0936-413c-a77a-cae077874dde.png">
<img width="1387" alt="Screen Shot 2021-10-11 at 7 20 32 PM" src="https://user-images.githubusercontent.com/35894540/136781865-2f959091-6265-43d0-bef4-dd70e9a842f1.png">

is that means I should specify the concrete IP address for "--sn-ip" field?

all the variables and init environment is correct:
<img width="1385" alt="Screen Shot 2021-10-11 at 7 27 38 PM" src="https://user-images.githubusercontent.com/35894540/136782684-931f33a5-af58-4f74-aa78-45952775a090.png">
<img width="1383" alt="Screen Shot 2021-10-11 at 7 28 04 PM" src="https://user-images.githubusercontent.com/35894540/136782717-bc03d36d-916a-4155-9a82-3581565bf0c2.png">

can you give me some suggestions? thank u!

**Comment by DoLNw** [comment]
hello, I still didn't find the solutions, can anyone give me some advice?

**Comment by RadhakrishnaJ** [comment]
> hello, I still didn't find the solutions, can anyone give me some advice?

Hello, Sorry we could not find any specific root cause yet. 

Could you please fill out below template and also provide the loggers of other containers also?
[Issue template](https://github.com/HewlettPackard/swarm-learning/blob/master/.github/ISSUE_TEMPLATE/swam-learning-issue-reporting-template.md)

**Comment by zl8255** [comment]
I also have the same problem, but after a test, I found that using "sudo" can solve this problem. You can try `sudo. / MNIST keras / bin / init workspace - E $example - I $APLS_ IP -d $WORKSPACE_ DIR`

**Comment by RadhakrishnaJ** [comment]
@DoLNw - Please confirm are you still blocked on this issue? If so, please provide issue template as requested above.

**Comment by RadhakrishnaJ** [comment]
Did not get any response from user on further loggers. Requesting user to create new issue and attach issue template for the issue.
